src/domains/chart/utils/get-chart-url-options.ts

```diff
@@ -0,0 +1,28 @@
+import { Attributes } from "domains/chart/utils/transformDataAttributes"
+import { chartLibrariesSettings } from "domains/chart/utils/chartLibrariesSettings"
+
+export const getChartURLOptions = (
```

Comment:
The string concat into a variable is a valid technique, maybe not worth to redo this function to be fancier/using templates with function calls to get specific string parts. Still pointed out, consider as a nitpick.

Reply:
i agree! i left it like this because how it was initially working in old dashboard.js, definitely something which can be shortened and more readable